app/src/main/java/org/squidfish/tuplenback/models/GameViewModel.kt

```diff
@@ -29,14 +30,24 @@ private const val TAG = "GameViewModel"
  *
  * @see[AppEvent]
  */
-class GameViewModel : ViewModel() {
+class GameViewModel(private val repository: LocalStorageRepository) : ViewModel() {
     private val _gameState = MutableStateFlow(GameState())
     val gameState: StateFlow<GameState> = _gameState.asStateFlow()
 
     private var timerJob: Job? = null
 
     private val gameEngines = mutableMapOf<GameModule, GameEngine>()
-    var game: Game = Game.None // TODO: load game played on last session
+    var game: Game? = null
+
+    init {
+        viewModelScope.launch {
+            val prevGame = repository.getRecent()?.gameType
+
+            if (prevGame != null) {
+                game = prevGame
+            }
```

Comment:
`prevGame?.let { game = it }` (just a personal preference, I know yours is smart cast)